egs/wsj/s5/steps/nnet3/chain2/compute_preconditioning_matrix.sh

```diff
@@ -76,7 +76,7 @@ if [ $stage -le 2 ]; then
         $ldafolder/lda.mat $ldafolder/lda_stats || exit 1
 
     rm $ldafolder/lda_stats
-    ln -rs $ldafolder/lda.mat $ldafolder/configs/lda.mat
+    ln -rfs $ldafolder/lda.mat $ldafolder/configs/lda.mat
```

Comment:
can we remove the pre-existing option 'r'?   It isn't present on mac.

Reply:
Done